main.go

```diff
@@ -29,8 +28,5 @@ func main() {
 	//Get database connection
 	database.GetConnection()
 
-	if len(os.Args) > 1 && os.Args[1] == "playground" {
```

Comment:
Changed to use an environment variable instead of command line argument to be consistent with other runtime options.